### PR TITLE
Upgrade to polymer 3

### LINKED ISSUE #34: Polymer 3 Support
When will you be rolling out Polymer 3 support for this component? It would be nice to have this component on npm.

### PR DESCRIPTION
**Pull request by cristinecula**
Fixes #34

### CODE REVIEW

**Comment by cristinecula** [comment]
Depends on:
https://github.com/Neovici/paper-autocomplete/pull/10
https://github.com/Neovici/cosmoz-i18next/pull/24

File: paper-autocomplete-chips.js [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+ * `--paper-autocomplete-chips-suggestions-width` | width of the suggestions container |
+ * `--paper-autocomplete-chips-suggestions-position` | position of the suggestions container | `absolute`
+ *
+ * @demo demo/index.html
+ * @polymer
+ * @customElement
+ * @appliesMixin translatable
+ */
+class PaperAutocompleteChips extends translatable(PolymerElement) {
+	static get template() {
```
Comment (nomego) [inline paper-autocomplete-chips.js]:
```suggestion
	static get template() { //eslint-disable-line max-lines-per-function
```

File: paper-autocomplete-chips.js [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+				</paper-item>
+				</template>
+		</paper-autocomplete>
+`;
+	}
+
+	static get is() {
+		return 'paper-autocomplete-chips';
+	}
+	static get properties() {
```
Comment (nomego) [inline paper-autocomplete-chips.js]:
```suggestion
	static get properties() { //eslint-disable-line max-lines-per-function
```

**Comment by cristinecula** [comment]
@nomego @Wurper ready for release!

**Pull request merged**